src/main/deltachat/login.js

```diff
@@ -48,13 +48,13 @@ module.exports = class DCLoginController extends SplitOut {
         this._controller.configuring = false
         this._controller.emit('ready', this._controller.credentials)
         log.info('dc_get_info', dc.getInfo())
-        render()
+        sendStateToRenderer()
       }
-      if (!this._dc.isConfigured()) {
+      if (!this._dc.isConfigured() || updateConfiguration) {
         this._dc.once('ready', onReady)
         this._controller.configuring = true
         this._dc.configure(addServerFlags(credentials))
```

Comment:
Here I would expect, that the credentials are updated in the database but they are not...??

Reply:
configure() function three lines below the change calls set_config.

Reply:
Yes but if I test it I don't get the updated values in the database...??